modin/engines/ray/cudf_on_ray/frame/partition_manager.py

```diff
@@ -29,6 +29,7 @@
 
 GPU_MANAGERS = []
 
+
```

Comment:
is this for formatting?

Reply:
Yea I believe this is from running the black formatter.

Reply:
okok